windows/test-app.ts

```diff
@@ -0,0 +1,300 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+const windowsDir = 'windows';
+const reactNativeModulePath = findClosestPathTo(path.join('node_modules', 'react-native-windows'));
+const testAppNodeModulePath = findClosestPathTo(path.join('node_modules', 'react-native-test-app'));
+const srcRootPath = path.join(testAppNodeModulePath, windowsDir);
+const destPath = path.resolve('');
+
+
+function findClosestPathTo(fileOrDirName: string): string {
+    var basePath = path.resolve('');
+    var rootDirectory = basePath.split(path.sep)[0] + path.sep;
```

Comment:
Prefer to use `const` or `let`. `var`s aren't properly scoped.
```suggestion
    let basePath = path.resolve('');
    const rootDirectory = basePath.split(path.sep)[0] + path.sep;
```